lib/grack.rb

```diff
@@ -1,5 +1,10 @@
 require "grack/bundle"
-
+require 'zlib'
+require 'rack/request'
+require 'rack/response'
+require 'rack/utils'
+require 'time'
+require 'grack/git'
```

Comment:
This is a requirement of `Grack::Server`, not `Grack` plain, so please move it to `grack/server.rb`